Recent floating point math not compiling with tcc

Fiwix doesn't compile with tcc now, ending with this error message:
```
tcc: error: undefined symbol '__fixdfdi'
make: *** [Makefile:70: all] Error 1
```

This is a result of adding floating point math recently using PAGE_HASH_PERCENTAGE.
The floating point math can be avoided with a simple change like I've pasted below. It's not the greatest patch so if you have any suggestions let me know. This could also be changed with `#ifdef __TINYC__`. If you have a preference and/or would like me to submit a PR please let me know.

```
diff --git a/include/fiwix/config.h b/include/fiwix/config.h
index 9c55834..ff87afb 100644
--- a/include/fiwix/config.h
+++ b/include/fiwix/config.h
@@ -16,7 +16,7 @@
 #define NR_FLOCKS              (NR_PROCS * 5)  /* max. number of flocks */

 #define FREE_PAGES_RATIO       5       /* % minimum of free memory pages */
-#define PAGE_HASH_PERCENTAGE   0.1     /* % of hash buckets relative to the
+#define PAGE_HASH_PER_10K       10     /* % of % of hash buckets relative to the
                                           number of physical pages */
 #define BUFFER_PERCENTAGE      100     /* % of memory for buffer cache */
 #define BUFFER_HASH_PERCENTAGE 10      /* % of hash buckets relative to the
diff --git a/mm/memory.c b/mm/memory.c
index 2addf61..56a3a00 100644
--- a/mm/memory.c
+++ b/mm/memory.c
@@ -450,7 +450,7 @@ void mem_init(void)
 #endif /* CONFIG_KEXEC */

        /* the last one must be the page_table structure */
-       n = (kstat.physical_pages * PAGE_HASH_PERCENTAGE) / 100;
+       n = (kstat.physical_pages * PAGE_HASH_PER_10K) / 10000;
        n = MAX(n, 1);  /* 1 page for the hash table as minimum */
        n = MIN(n, 16); /* 16 pages for the hash table as maximum */
        page_hash_table_size = n * PAGE_SIZE;
```

--- a/include/fiwix/config.h
+++ b/include/fiwix/config.h
@@ -16,7 +16,7 @@
 #define NR_FLOCKS		(NR_PROCS * 5)	/* max. number of flocks */
 
 #define FREE_PAGES_RATIO	5	/* % minimum of free memory pages */
-#define PAGE_HASH_PERCENTAGE	0.1	/* % of hash buckets relative to the
+#define PAGE_HASH_PER_10K	10	/* % of % of hash buckets relative to the
 					   number of physical pages */
 #define BUFFER_PERCENTAGE	100	/* % of memory for buffer cache */
 #define BUFFER_HASH_PERCENTAGE	10	/* % of hash buckets relative to the

--- a/mm/memory.c
+++ b/mm/memory.c
@@ -450,7 +450,7 @@ void mem_init(void)
 #endif /* CONFIG_KEXEC */
 
 	/* the last one must be the page_table structure */
-	n = (kstat.physical_pages * PAGE_HASH_PERCENTAGE) / 100;
+	n = (kstat.physical_pages * PAGE_HASH_PER_10K) / 10000;
 	n = MAX(n, 1);	/* 1 page for the hash table as minimum */
 	n = MIN(n, 16);	/* 16 pages for the hash table as maximum */
 	page_hash_table_size = n * PAGE_SIZE;

